--- build/linux/Makefile.inOLD	2015-06-11 15:48:59.148117052 -0500
+++ build/linux/Makefile.in	2015-06-11 15:56:39.994803653 -0500
@@ -253,6 +253,9 @@
 	cd ../../python; CPPFLAGS="${PYCPPFLAGS}" LDFLAGS="${PYLDFLAGS}" $(PYTHON) builder.py install \
 	--install-base=./finalbuild --install-headers=./finalbuild --install-purelib=./finalbuild \
 	--install-platlib=./finalbuild --install-scripts=./finalbuild --install-data=./finalbuild
+
+python-root-install: libastra.la
+	cd ../../python; CPPFLAGS="${PYCPPFLAGS}" LDFLAGS="${PYLDFLAGS}" $(PYTHON) builder.py install
 endif
 
 
